src/components/text-editor/utils/html-converter.ts

```diff
@@ -1,14 +1,27 @@
 import { ContentTypeConverter } from './content-type-converter';
 import { EditorView } from 'prosemirror-view';
+import { markdownToHTML } from '../../markdown/markdown-parser';
+import { CustomElementDefinition } from 'src/interface';
```

Comment:
This import might be problematic if I remember correctly, it might mess up the generated exported types. A relative import should work, e.g. `../../../interface` (I might have gotten the number of levels incorrect :smile: )

Reply:
⚡ https://github.com/Lundalogik/lime-elements/commit/5e0a2c42030cf48ee8b9477a026bf95a0f06f78a